infinite-list/infinite-list.tag

```diff
@@ -0,0 +1,25 @@
+<infinite-list onscroll={ onscroll }>
+
+  <h3>{ title }</h3>
+  
+  <list-item each={ items_to_render }></list-item>
+
```

Comment:
Could you use `<script>` tag here? Basically we encourage to put the JavaScript into `<script>` in this repo, basically.